ivy_tests/test_ivy/test_frontends/test_mindspore/test_ops/test_function/test_mindspore_nn_func.py

```diff
@@ -360,3 +360,49 @@
 #         count_include_pad=count_include_pad,
 #         divisor_override=None,
 #     )
+
+
+# @handle_frontend_test(
+#     fn_tree="mindspore.ops.function.nn_func.smooth_l1_loss",
+#     dtype_and_x=helpers.dtype_and_values(
+#         available_dtypes=helpers.get_dtypes("float"),
+#         num_arrays=2,
+#         allow_inf=False,
+#         shared_dtype=True,
+#     ),
+#     size_average=st.booleans(),
+#     reduce=st.booleans(),
+#     reduction=st.sampled_from(["none", "mean", "sum"]),
```

Comment:
I'm not sure where these parameters come from, since they're not part of the function being tested

Reply:
i think we can remove this..

Reply:
reduction can be removed as mindspore ,doesn't have that support..